packages/flutter/lib/src/widgets/ticker_provider.dart

```diff
@@ -92,24 +109,73 @@ class TickerMode extends StatefulWidget {
   ///
   /// In the absence of a [TickerMode] widget, this function returns a
   /// [ValueListenable], whose [ValueListenable.value] is always true.
+  @Deprecated(
+    'Use TickerMode.getValuesNotifier to get both enabled and forceFrames. '
+    'This feature was deprecated after v3.35.0-0.0.pre.',
+  )
   static ValueListenable<bool> getNotifier(BuildContext context) {
     final _EffectiveTickerMode? widget = context
         .getInheritedWidgetOfExactType<_EffectiveTickerMode>();
     return widget?.notifier ?? const _ConstantValueListenable<bool>(true);
   }
 
+  /// Returns the effective ticker mode values for this subtree and establishes
+  /// a dependency on the surrounding [TickerMode].
+  static TickerModeData valuesOf(BuildContext context) {
+    final _EffectiveTickerMode? widget = context
+        .dependOnInheritedWidgetOfExactType<_EffectiveTickerMode>();
+    return widget?.values ?? const TickerModeData(enabled: true, forceFrames: false);
+  }
+
+  /// Obtains a [ValueListenable] of [TickerModeData] from the surrounding [TickerMode]
+  /// without establishing a dependency.
+  static ValueListenable<TickerModeData> getValuesNotifier(BuildContext context) {
+    final _EffectiveTickerMode? widget = context
+        .getInheritedWidgetOfExactType<_EffectiveTickerMode>();
+    return widget?.valuesNotifier ??
+        const _ConstantTickerModeDataListenable(TickerModeData(enabled: true, forceFrames: false));
+  }
+
+  /// Creates a [TickerMode] that overrides the ambient ticker mode values.
+  ///
+  /// The given `enabled` and `forceFrames` override the ambient values when not null;
+  /// otherwise the ambient values are preserved.
+  static Widget merge({Key? key, bool? enabled, bool? forceFrames, required Widget child}) {
+    return Builder(
+      builder: (BuildContext context) {
+        final _EffectiveTickerMode? parent = context
+            .dependOnInheritedWidgetOfExactType<_EffectiveTickerMode>();
+        final bool parentEnabled = parent?.enabled ?? true;
+        final bool parentForce = parent?.forceFrames ?? false;
```

Comment:
If you do add `TickerModeData.fallback`, let's use it here as well:

```suggestion
        final bool parentEnabled = parent?.enabled ?? TickerModeData.fallback.enabled;
        final bool parentForce = parent?.forceFrames ?? TickerModeData.fallback.forceFrames;
```